### DISCUSSION TITLE
Adding the search box to the windows 10 taskbar

### DISCUSSION THREAD

**Discussion by Mqlb** [valinet/ExplorerPatcher#1335] [Ideas]
I was thinking of adding the search box to the Windows 10 taskbar ![image](https://user-images.githubusercontent.com/122155865/232320721-8743d26a-4073-4192-8d6c-988dce7cbc80.jpeg)and I have a picture of how I think it can look like![image](https://user-images.githubusercontent.com/122155865/232320692-8d8b880c-df1f-4cdc-b82d-e23b6de335db.jpeg)

**Comment by Alex423243** [comment]
maybe it can look like the windows 10 search box instead
